### ISSUE TITLE
v0.4 - vendor/autoload.php: failed to open stream / Uncaught Error: Class 'Mustache_Engine' not found

### ISSUE THREAD

**Issue by bramwillemse** [baardbaard/bb-twitterfeed#34]
When I try to activate v0.4, I get this error: 

```
Warning: require_once(vendor/autoload.php): failed to open stream: No such file or directory in <PATH>/web/app/plugins/bb-twitterfeed/bb-twitterfeed.php on line 30

Fatal error: require_once(): Failed opening required 'vendor/autoload.php' (include_path='.:') in <PATH>/web/app/plugins/bb-twitterfeed/bb-twitterfeed.php on line 30
```

Off course the fix should be performing `composer dump-autoload`, but ... (see next comment)

**Comment by bramwillemse** [comment]
When I perform a `composer dump-autoload` in the plugin folder (which should fix it, right?) I get the following error when trying to activate the plugin: 

```
Fatal error: Uncaught Error: Class 'Mustache_Engine' not found in /<PATH>/web/app/plugins/bb-twitterfeed/classes/class-bb-twitterfeed.php:46 
Stack trace: 
#0 /<PATH>/web/app/plugins/bb-twitterfeed/bb-twitterfeed.php(40): Twitterfeed\Twitterfeed->__construct() 
#1 /<PATH>/web/wp/wp-admin/includes/plugin.php(1943): include('/<PATH>...') 
#2 /<PATH>/web/wp/wp-admin/plugins.php(164): plugin_sandbox_scrape('bb-twitterfeed/...') 
#3 /<PATH>/.composer/vendor/laravel/valet/server.php(106): require('/<PATH>...') 
#4 {main} thrown in /<PATH>/web/app/plugins/bb-twitterfeed/classes/class-bb-twitterfeed.php on line 46
```

**Comment by stefthoen** [comment]
Did you run `composer install`? It seems to be missing the Mustache package.

**Comment by bramwillemse** [comment]
@baardbaard Yes. It's a fresh install of the plugin!

**Comment by stefthoen** [comment]
I'm getting the same errors if I don't run `composer install`. Could you try again but first delete everything? Maybe even run `composer clearcache` before you run `composer install`. You don't get any errors when running `composer install`?

**Comment by bramwillemse** [comment]
I already did all that. But tried it once more. And again I'll get the first error:

```
Warning: require_once(vendor/autoload.php): failed to open stream: No such file or directory in /<PATH>/web/app/plugins/bb-twitterfeed/bb-twitterfeed.php on line 30

Fatal error: require_once(): Failed opening required 'vendor/autoload.php' (include_path='.:') in /<PATH>/web/app/plugins/bb-twitterfeed/bb-twitterfeed.php on line 30
```

**Comment by stefthoen** [comment]
Should be fixed in `0.5`. Check out the new install instructions because some things probably changed. Let me know if this fixes it for you.
